ingest-app/src/migrations/009_update_bulk_status_message.clj

```diff
@@ -0,0 +1,20 @@
+(ns migrations.009-update-bulk-status-message
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 9."
+  []
+  (println "migrations.009-update-bulk-status-message up...")
+  (j/db-do-commands (config/db)
+                    "ALTER TABLE bulk_update_coll_status MODIFY
+                     (status_message VARCHAR2(4000))"))
+
+(defn down
+  "Migrates the database down to version 8."
+  []
+  (println "migrations.009-update-bulk-status-message down...")
+  (j/db-do-commands (config/db) "TRUNCATE TABLE bulk_update_coll_status")
```

Comment:
I just saw this statement. We shouldn't delete data from the tables.

What we might have wanted to do was an update statement to every row to replace the status_message with up to the first 255 characters of the status message. But you cannot reduce the length of a column so this would need to be done by creating a new temporary column, dropping the old column and then renaming the temporary column to the original column.

After thinking that through It would be a crazy amount of work to do for a down migration. In this particular case we shouldn't do anything in the down migration (leave the column size at 4000).

Reply:
Cool, thats they way I had it before.